libs/components/src/next/form/checkbox/vl-checkbox.component.ts

```diff
@@ -0,0 +1,162 @@
+import { checkboxStyle } from '@domg/govflanders-style/component';
+import { baseStyle, resetStyle } from '@domg/govflanders-style/common';
+import { CSSResult, html, PropertyDeclarations, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { vlElementsStyle } from '@domg-wc/elements';
+import { classMap } from 'lit/directives/class-map.js';
+import checkboxUigStyle from './vl-checkbox.component.uig-css';
+import { FormControl } from '../form-control/FormControl';
+
+export const CheckboxDefaults = {
+    value: '',
+    block: false,
+    isSwitch: false,
+    checked: false,
+    disabled: false,
+    error: false,
+    label: 'label',
+    name: '',
+    value: '',
+};
+
+@customElement('vl-checkbox-next')
+export class VlCheckboxComponent extends FormControl {
+    private value = CheckboxDefaults.value;
+    private checked = CheckboxDefaults.checked;
+    private isSwitch = CheckboxDefaults.isSwitch;
+    private initialValue: string | boolean = '';
+    private initialCheckedValue: boolean | undefined = undefined;
+    private touched = false;
+    static get styles(): (CSSResult | CSSResult[])[] {
+        // TODO: why do we need to add vlElementsStyle here?
+        return [resetStyle, baseStyle, vlElementsStyle, checkboxStyle, checkboxUigStyle];
+    }
+
+    static get properties(): PropertyDeclarations {
+        return {
+            checked: { type: Boolean, reflect: true },
+            value: { type: String, attribute: 'value', reflect: true },
+            isSwitch: { type: Boolean, attribute: 'switch', reflect: true },
+        };
+    }
+
+    constructor() {
```

Comment:
Dit lijkt me overbodig aangezien de default values al gezet worden in de FormControl en de checkbox geen andere default values voorziet.